apps/frontend/src/network/teams/TeamProfile.tsx

```diff
@@ -35,8 +41,11 @@ const TeamProfile: FC<Record<string, never>> = () => {
   if (team) {
     return (
       <Frame title={team.displayName}>
-        <TeamProfilePage teamListElementId={teamListElementId} {...team}>
-          <Switch>
+        <Switch>
+          <Route path={path + route({ teamId }).createOutput.template}>
```

Comment:
This should probablly be wrapped in a Frame. Frames allow you to set the page title and provide the highest level error boundary which should be in scope imo